### DISCUSSION TITLE
Create custom color scheme

### DISCUSSION THREAD

**Discussion by melarenina** [plouc/nivo#1599] [Q&A] [answered]
Hi,

I haven't found anything regarding this topic, but is there a way to create a custom color scheme for my project? 

And also, is there a way to make this kind of styling in my charts, alternating two colors, dynamically?  without having to pass each value for each individual data? 

![chart](https://user-images.githubusercontent.com/38476041/122407276-5a979800-cf36-11eb-882e-a92d5fcad399.png)

Thank you!

**Comment by wyze** [comment, marked as answer]
You can pass a custom function to the `colors` property and achieve your goal: https://codesandbox.io/s/tender-shtern-pfk6x?file=/src/App.tsx

We also have a guide on theming which might help with your color scheme question. https://nivo.rocks/guides/theming

**Reply by stephent** [reply]
Is the custom function formally documented anywhere? For example, when I look at https://nivo.rocks/bar/,  the only documented allowed type for `colors` is `OrdinalColorScaleConfig`. There's also no mention of this on the dedicated colors guide page (https://nivo.rocks/guides/colors).
